server/statistics/hot_peer.go

```diff
@@ -110,3 +156,30 @@ func (stat *HotPeerStat) Clone() *HotPeerStat {
 	ret.rollingKeyRate = nil
 	return &ret
 }
+
+func (stat *HotPeerStat) isHot(thresholds [dimLen]float64) bool {
+	return stat.rollingByteRate.isHot(thresholds) || stat.rollingKeyRate.isHot(thresholds)
+}
+
+func (stat *HotPeerStat) clearLastAverage() {
+	stat.rollingByteRate.clearLastAverage()
+	stat.rollingKeyRate.clearLastAverage()
+}
+
+// Log is used to output some info
+func (stat *HotPeerStat) Log(str string, level func(msg string, fields ...zap.Field)) {
```

Comment:
looks wired

Reply:
Well, I will remove this log temporarily and will add after usb log merged